cmd/updater/update.sh

```diff
@@ -695,15 +677,15 @@ if ! $DRYRUN; then
         fail_and_exit "Error installing new files"
     fi
 
-    for DD in "${DATADIRS[@]}"; do
+    for DD in ${DATADIRS[@]}; do
         if ! install_new_data "${DD}"; then
             fail_and_exit "Error installing data files into ${DD}"
         fi
     done
 
     copy_genesis_files
 
-    for DD in "${DATADIRS[@]}"; do
+    for DD in ${DATADIRS[@]}; do
```

Comment:
Does this work without the quotes? I'm not positive bash will interpret the bash array correctly without them

Reply:
This was John's change. His PR notes are here: https://github.com/algobarb/go-algorand/pull/2
I'm running it again to make sure this works as expected.

Reply:
looks like they don't matter actually in a simple example, so i think this should be ok. Removing them might be unnecessary risk unless you had a reason to remove them though.